salt/states/cloud.py

```diff
@@ -444,8 +445,8 @@ def volume_detached(name, server_name=None, provider=None, **kwargs):
     if name in volumes and not volumes[name]['attachments']:
         volume = volumes[name]
         ret['comment'] = (
-            'Volume {name} is not currently attached to anything.'
-        ).format(**volumes[name])
+            'Volume {0} is not currently attached to anything.'
+        ).format(name, **volumes[name])
```

Comment:
If `{0}` is `name` then we don't need `**volumes[name]` in the format arguments here.